vcstool/commands/compare.py

```diff
@@ -1,80 +1,595 @@
 import argparse
-from cgitb import reset
-from distutils.util import execute
-from gettext import find
-from prettytable import PrettyTable
 import sys
-from unittest import result
-from vcstool import clients
-from vcstool.commands.branch import BranchCommand
-from vcstool.crawler import find_repositories
+import os
+import abc
+from typing import Any, Dict, List, Optional
+from enum import IntEnum, IntFlag, auto
+
+import prettytable as pt
 
-from vcstool.executor import ansi, execute_jobs, generate_jobs, output_repositories
+from vcstool.crawler import find_repositories
+from vcstool.executor import ansi, execute_jobs, generate_jobs
 from vcstool.commands.import_ import get_repositories
 from vcstool.streams import set_streams
+from vcstool.outputs import CompareOutput
 
 from .command import add_common_arguments
 from .command import Command
 
 
 class CompareCommand(Command):
 
-    command = 'compare'
```

Comment:
Nit: was trying to follow quote format from other _commands_ and therefore used single quotes.

Reply:
Yeah I figured you'd notice this lol. I actually took the liberty of using `black` formatter for this file, which defaults to double-quotes instead of single-quotes. I can add a `pyproject.toml` to this project and change to single-quotes.

Also I avoided applying formatting to existing project files to keep the diff smaller.